### PR TITLE
Link, Nav, and Button Styling Updates

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: src/styles/_header.scss [hunk 1]
```diff
@@ -171,6 +179,9 @@
           );
           &:hover {
             border: none;
+            &::after {
+              border: 3px solid color('secondary');
+            }
```
Comment (saifulislamdev) [inline src/styles/_header.scss:184]: Which lines make the page not jump on a button hover?

File: src/styles/_header.scss [hunk 2]
```diff
@@ -294,46 +299,48 @@
...
-          'base-dark'
-        );
         margin: 0 units('105');
         &:hover {
           padding-top: 0;
         }
       }
     }
   }
+  // to indicate which language is in use or active, there is a bar on one side on one side of the language button
```
Comment (saifulislamdev) [inline src/styles/_header.scss]: `on one side on one side`

Also, it's a good idea to say where the language button is (language selector).

File: src/styles/_header.scss [hunk 3]
```diff
@@ -294,46 +299,48 @@
...
+  // left-to-right languages active border
+  .banner__language-selector-item--ltr.banner__language-selector-item--active {
```
Comment (saifulislamdev) [inline src/styles/_header.scss:332]: Does this count as 2 classes or 1 class?

File: src/styles/_header.scss (hunk 1 above)
Comment (aannyyaa) [reply src/styles/_header.scss:184]: Setting `$border-color` and `$border-color-hover` in the `button-theme` mixins in the global styles file

File: src/styles/_header.scss (hunk 2 above)
Comment (aannyyaa) [reply src/styles/_header.scss]: Not sure how I typed that twice! And I added a bit more description here and in other lines related to the language buttons

File: src/styles/_header.scss (hunk 3 above)
Comment (aannyyaa) [reply src/styles/_header.scss:332]: I think 2 but they both apply

Comment (saifulislamdev) [reply src/styles/_header.scss:332]: Interesting.

**Review by saifulislamdev — APPROVED** [review]

**Pull request merged**